client/context/context.go

```diff
@@ -97,12 +100,15 @@ func NewCLIContextWithFrom(from string) CLIContext {
 		FromName:      fromName,
 		Indent:        viper.GetBool(flags.FlagIndentResponse),
 		SkipConfirm:   viper.GetBool(flags.FlagSkipConfirmation),
+		SecretStore:   viper.GetBool(flags.FlagSecretStore),
 	}
 }
 
 // NewCLIContext returns a new initialized CLIContext with parameters from the
 // command line using Viper.
-func NewCLIContext() CLIContext { return NewCLIContextWithFrom(viper.GetString(flags.FlagFrom)) }
+func NewCLIContext(input io.Reader) CLIContext {
```

Comment:
This API should not change. Rather, a user should call `NewCLIContext().WithInput(reader)...`

Reply:
same reasoning as https://github.com/cosmos/cosmos-sdk/pull/4754#discussion_r314135314

Reply:
See response; I do not believe this API should change solely due to Keybase requirements.